plugins/@grouparoo/calculated-property/src/lib/propertyOptions.ts

```diff
@@ -0,0 +1,24 @@
+import { Property, PropertyOptionsMethod } from "@grouparoo/core";
+
+export const propertyOptions: PropertyOptionsMethod = async () => [
+  {
+    key: "requiredProperties",
+    required: true,
+    description:
+      "Grouparoo Property IDs for all properties used in customFunction",
+    type: "lix  st",
+    options: async () => {
+      return undefined;
+    },
```

Comment:
This might require some updates in core to sort out

Reply:
Is this referring to the type typo?

Reply:
... probably! I think it's already fixed up